extensions/arc/src/ui/dashboards/postgres/postgresConnectionStringsPage.ts

```diff
@@ -87,6 +95,8 @@ export class PostgresConnectionStringsPage extends DashboardPage {
 	}
 
 	private handleServiceUpdated() {
+		this.connectionStringsLoading.loading = true;
```

Comment:
I'm wondering if we need to set loading = true here.  The refresh and getConnectionStrings calls below don't do anything long running like call azdata, they just update the UI.  And since there's no refresh button on this page, the only load is the initial one.

Reply:
Yeah that's probably not necessary - this is going to be pretty quick most of the time. We mostly just care about having it initially be loading and then setting loading to false once we get updated service values.